packages/terra-application-name/examples/Index.jsx

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import PropsTable from 'terra-props-table';
+import Markdown from 'terra-markdown';
+import ReadMe from 'terra-application-name/docs/README.md';
+import { version } from 'terra-application-name/package.json';
+
```

Comment:
hmm. Right now this will want you to pull in the markdown & props-table as devDependencies. These will be provided by terra-site once pulled in. Maybe just comment out for now to keep things clean. @bjankord @mjhenkes Thoughts?